x-pack/plugin/esql/qa/testFixtures/src/main/resources/eval.csv-spec

```diff
@@ -356,22 +356,43 @@ FROM sample_data
 
 docsLength
 // tag::length[]
-FROM employees
-| KEEP first_name, last_name
-| EVAL fn_length = LENGTH(first_name)
+FROM airports
+| WHERE country == "India"
+| KEEP city
+| EVAL fn_length = LENGTH(city)
 // end::length[]
-| SORT first_name
+| SORT city
 | LIMIT 3
 ;
 
 // tag::length-result[]
-first_name:keyword   | last_name:keyword   | fn_length:integer
-Alejandro      |McAlpine       |9              
-Amabile        |Gomatam        |7              
-Anneke         |Preusig        |6
+city:keyword  | fn_length:integer
+Agwār         | 5
+Ahmedabad     | 9
+Bangalore     | 9
 // end::length-result[]
 ;
 
+docsByteLength
+required_capability: fn_byte_length
+// tag::byteLength[]
+FROM airports
+| WHERE country == "India"
+| KEEP city
+| EVAL fn_byte_length = BYTE_LENGTH(city)
+// end::byteLength[]
+| SORT city
+| LIMIT 3
+;
+
+// tag::byteLength-result[]
+city:keyword  | fn_byte_length:integer
+Agwār         | 6
```

Comment:
It would be nice to have two columns, one for length and one for byte_length. Having this in only the byte_length example is sufficient, we don't need to do the same for the length and bit_length examples.

Reply:
For symmetry's sake, it think it should be the same for `bit_length` as well, i.e., have both `length` and `byte_length`. The external documentation shouldn't rely that you read `byte_length` before :)